docs/ref/forms/fields.txt

```diff
@@ -1260,13 +1260,47 @@ method::
 
     .. attribute:: queryset
 
-        Same as :class:`ModelChoiceField.queryset`.
+        A ``QuerySet`` of model objects from which the choices for the field
+        are derived and which is used to validate the user's selection. It's
+        evaluated when the form is rendered.
 
     Takes one optional argument:
 
     .. attribute:: to_field_name
 
-        Same as :class:`ModelChoiceField.to_field_name`.
+        This optional argument is used to specify the field to use as the value
+        of the choices in the field's widget. Be sure it's a unique field for
+        the model, otherwise the selected value could match more than one
+        object. By default it is set to ``None``, in which case the primary key
+        of each object will be used. For example::
+
+            # No custom to_field_name
+            field1 = forms.ModelMultipleChoiceField(queryset=...)
+
+        would yield:
+
+        .. code-block:: html
+
+            <select id="id_field1" name="field1" multiple>
+            <option value="obj1.pk">Object1</option>
```

Comment:
IMO - we should indent the option just like [W3 school](https://www.w3schools.com/tags/tag_option.asp) is showing

Reply:
I'm okay either way? @carltongibson can you clarify if you have a preference here? This change was created based on your comment in PR #12088.

Reply:
Hey @jdufresne. No real preference... I guess one might indent it for readability but :shrug:

Reply:
Ok, in that case I think I'll keep the established convention here and elsewhere in the docs and leave as-is. If others have opinions on this, please speak up and I can go ahead.